### DISCUSSION TITLE
Why focus state layer were removed in all components?

### DISCUSSION THREAD

**Discussion by VietNguyenQuoc** [material-components/material-web#5566] [Q&A]
Hi @asyncLiz, I see that focus state layer were removed in all components in https://github.com/material-components/material-web/blob/main/CHANGELOG.md#100-pre10-2023-06-05.
May I ask what was the decision then? Any plan to re-add in the future?

**Comment by asyncliz** [comment]
Focus ripples were removed in the Material spec in favor of focus rings, which are more accessible. Ripples only indicate hover and pressed states now.

I don't think material.io has been updated with the new focus ring specs unfortunately. There are no plans to add it back to the spec.
